toml.abnf

```diff
@@ -117,14 +117,19 @@ underscore = %x5F                  ; _
 
 ;; Float
 
-float = integer ( frac / ( frac exp ) / exp )
+float =  integer ( frac / ( frac exp ) / exp )
+float =/ non-finite
```

Comment:
Maybe call them "named float values"?

Reply:
From a bit of googling, it seems like "special values" is the preferred nomenclature. How does that strike you?

Reply:
Works well. :)